src/controllers/dynakube/version/updater_test.go

```diff
@@ -344,11 +345,11 @@ func TestUpdateVersionStatus(t *testing.T) {
 		expectedDigest := "sha256:7ece13a07a20c77a31cc36906a10ebc90bd47970905ee61e8ed491b7f4c5d62f"
 		expectedID := expectedRepo + ":tag@" + expectedDigest
 		target := status.VersionStatus{}
-		boomFunc := func(_ context.Context, imagePath string, _ *dockerconfig.DockerConfig) (ImageVersion, error) {
+		boomFunc := func(_ context.Context, _ registry.Client, imagePath string, _ *dockerconfig.DockerConfig) (registry.ImageVersion, error) {
```

Comment:
```suggestion
		boomFunc := func(_ context.Context, _ registry.ImageGetter, imagePath string, _ *dockerconfig.DockerConfig) (registry.ImageVersion, error) {
```
or something like it.